src/app/pages/game-screen/game-screen.component.html

```diff
@@ -8,4 +8,3 @@
   <app-map class="map"></app-map>
   <app-chat class="chat"></app-chat>
 </div>
-<app-route-selector *ngIf="selectingRoutes"></app-route-selector>
```

Comment:
Why did you get rid of the route selector?

Reply:
Oh whoops. I removed it to make development easier. I didn't mean to commit that.